apps/data-service-catalog/components/data-service-form/components/contact-point-section.tsx

```diff
@@ -32,6 +33,15 @@ export const ContactPointSection = () => {
     });
   };
 
+  const getErrors = () => {
+    return lodashGet(errors, 'contactPoint');
```

Comment:
Hvorfor bruker du denne isteden for Formik sin?

Reply:
det jeg gjorde her var sterkt inspirert av hva @jeffreiffers gjorde her: https://github.com/Informasjonsforvaltning/catalog-frontend/blame/main/libs/ui/src/lib/formik-language-fieldset/index.tsx#L71-L78

Visste ikke at formik hadde funksjonalitet for det :)

Reply:
Hmm skjønner, bare lurte på om det var noe jeg burde gjøre også 😛 Finnes en errors i useFormikContext slik at man kan bruke errors.contactPoint men er litt usikker på hva som blir forskjellen her

Reply:
Ja det er å bare bruke errors param. Sjekk i concept-catalog. Brukt flere steder.

Reply:
Errors er kun et object så bruker lodash til å aksessere verdier basert på stien.

Reply:
Akkurat for language fields så bruker jeg dette for å lage en error summarize basert på flere felt.